### ISSUE TITLE
Lexrank for Single Document 

### ISSUE THREAD

**Issue by alkakhurana** [crabcamp/lexrank#35]
How can I use this implementation of Lexrank for summarizing a single document?
Attempting this produces the following error - 
ValueError: documents are not informative

Thanks in advance
Alka Khurana

**Comment by yungsinatra0** [comment]
Have you been able to find a solution for this? I've stumbled upon this issue as well.

**Comment by emnlp-cherry** [comment]
I do not think it is because of a single document. The issue happens when the bag of words is empty, it can be due to sentence tokenization error, such as considering "." as a sentence.

**Comment by ijaradat** [comment]
The issue is not the single document. The issue is that the code is old and using old python lib. you can fix summarizer.py using ntlk tokenization by replacing lines 117-119 with the following:             
```
for sentence in sent_tokenize(doc):
                words = word_tokenize(sentence)
                doc_words.update(words)
```
and do not forget to include sent_tokenize and word_tokenize from ntlk
